src/components/ui/cardWrapper.tsx

```diff
@@ -71,12 +75,22 @@ const CardWrapper = React.forwardRef<HTMLDivElement, CardWrapperProps>(
           )}
         </div>
         <div className={areaClasses}>
-          <AreaCard
-            id={area.id}
-            name={area.name}
-            visible={isAreaVisible}
-            toggleVisibility={toggleVisibility}
-          />
+          {/* Conditionally render AreaCard or AreaCardAssessment */}
+          {userLoggedIn ? (
+            <AreaCardAssessment
+              id={area.id}
+              name={area.name}
+              visible={isAreaVisible}
+              toggleVisibility={toggleVisibility}
+            />
+          ) : (
+            <AreaCard
+              id={area.id}
+              name={area.name}
+              visible={isAreaVisible}
+              toggleVisibility={toggleVisibility}
+            />
+          )}
         </div>
```

Comment:
This logic is not needed. You are displaying the two first buttons anyway, why have two components that share the same buttons? Always show AreaCard, and if the user is logged in, show the edit button as well. No need to create a component that is identical to AreaCard and augment it with the edit button.

If we wish to edit the info button for instance, we'll now have to do it twice, which is redundant.

Reply:
I see your point, let me change it again